src/main/java/school/hei/patrimoine/visualisation/swing/ihm/MainIHM.java

```diff
@@ -77,13 +77,26 @@ private void configureContentPane() {
     westPanel.setAlignmentY(TOP_ALIGNMENT);
     var westMargin = 5;
     westPanel.setBorder(createEmptyBorder(westMargin, westMargin, westMargin, westMargin));
+
     westPanel.add(selecteurPatrimoineIHM);
     westPanel.add(selecteurGrapheConfIHM);
     westPanel.add(selecteurPeriodeIHM);
     westPanel.add(fluxImpossiblesIHM);
+
+    var fluxJournaliersBox = new BoxLayout(fluxJournaliersIHM, BoxLayout.X_AXIS);
+    fluxJournaliersIHM.setLayout(fluxJournaliersBox);
+    fluxJournaliersIHM.setBorder(BorderFactory.createEmptyBorder(10, 5, 0, 5));
+
+    Dimension fixedWidth =
+        new Dimension(
+            fluxImpossiblesIHM.getPreferredSize().width,
+            fluxJournaliersIHM.getPreferredSize().height);
+    fluxJournaliersIHM.setPreferredSize(fixedWidth);
+    fluxJournaliersIHM.setMaximumSize(fixedWidth);
+    fluxJournaliersIHM.setMinimumSize(fixedWidth);
```

Comment:
put this block of code in a method named getFluxImpossiblesIHM

Reply:
done